packages/react-web-worker/src/test/hooks.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {mount} from '@shopify/react-testing';
+import {useWorker} from '../index';
+
+function mockCreate(): () => {} {
+  return () => {
+    return {
+      hello(friend: string) {
+        return `Hello, ${friend}`;
+      },
+    };
+  };
+}
+describe('useWorker', () => {
+  function MockComponent({create = mockCreate(), children}) {
```

Comment:
We should need to provide a type here (IIRC, TS was complaining in the editor about this prop). Yes, you will find that quilt's CI does not check types for tests, which is fairly shocking, I agree :P